README.md

```diff
@@ -229,13 +244,13 @@ auto-cpufreq should be run with with one of the following options:
 * version
     - Show currently installed version
 
-* [donate](#financial-donation)
+* [donate](https://github.com/AdnanHodzic/auto-cpufreq/#financial-donation)
     - To support the project
 
 * help
     - Shows all of the above options
 
-Running `auto-cpufreq --help` will print the same list of options as above. Read [auto-cpufreq modes and options](#auto-cpufreq-modes-and-options) for more details.
+Running `auto-cpufreq --help` will print the same list of options as above. Read [auto-cpufreq modes and options](https://github.com/AdnanHodzic/auto-cpufreq/#auto-cpufreq-modes-and-options) for more details.
```

Comment:
let's update this link to `[auto-cpufreq modes and options](#auto-cpufreq-modes-and-options)`